README.md

```diff
@@ -85,7 +85,7 @@ To install the Python version of the beautifier:
 ```bash
 $ pip install jsbeautifier
 ```
-
+Unlike the JavaScript version, the Python version can only reformat JavaScript. It does not work against HTML or CSS files.
```

Comment:
There's a `css-beautify` that also works and I believe is included in the Python package.